modules/installation-guide/examples/system-variables.adoc

```diff
@@ -29,23 +29,23 @@ pass:[<!-- vale off -->]
+ `+CHE_WORKSPACE_SIDECAR_DEFAULT__CPU__LIMIT__CORES+`,"`+-1+`","CPU limit default for each sidecar that has no CPU settings in the {prod-short} plug-in configuration. Specify either in floating point cores number, for example, `0.125`, or using the Kubernetes format, integer millicores, for example, `125m`. Value less or equal to 0 is interpreted as disabling the limit." 
+ `+CHE_WORKSPACE_SIDECAR_DEFAULT__CPU__REQUEST__CORES+`,"`+-1+`","CPU request default for each sidecar that has no CPU settings in the {prod-short} plug-in configuration. Specify either in floating point cores number, for example, `0.125`, or using the Kubernetes format, integer millicores, for example, `125m`." 
  `+CHE_WORKSPACE_SIDECAR_IMAGE__PULL__POLICY+`,"`+Always+`","Defines image-pulling strategy for sidecars. Possible values are: `Always`, `Never`, `IfNotPresent`. For any other value, `Always` is assumed for images with the `:latest` tag, or `IfNotPresent` for all other cases." 
  `+CHE_WORKSPACE_ACTIVITY__CHECK__SCHEDULER__PERIOD__S+`,"`+60+`","Period of inactive workspaces suspend job execution." 
- `+CHE_WORKSPACE_ACTIVITY__CLEANUP__SCHEDULER__PERIOD__S+`,"`+3600+`","The period of the cleanup of the activity table. The activity table can contain invalid or stale data if some unforeseen errors happen, like a server crash at a peculiar point in time. The default is to run the cleanup job every hour." 
+ `+CHE_WORKSPACE_ACTIVITY__CLEANUP__SCHEDULER__PERIOD__S+`,"`+3600+`","The period of the cleanup of the activity table. The activity table can contain invalid or stale data if some unforeseen errors happen, as a server failure at a peculiar point in time. The default is to run the cleanup job every hour." 
  `+CHE_WORKSPACE_ACTIVITY__CLEANUP__SCHEDULER__INITIAL__DELAY__S+`,"`+60+`","The delay after server startup to start the first activity clean up job." 
  `+CHE_WORKSPACE_ACTIVITY__CHECK__SCHEDULER__DELAY__S+`,"`+180+`","Delay before first workspace idleness check job started to avoid mass suspend if ws master was unavailable for period close to inactivity timeout." 
- `+CHE_WORKSPACE_CLEANUP__TEMPORARY__INITIAL__DELAY__MIN+`,"`+5+`","Period of stopped temporary workspaces cleanup job execution." 
- `+CHE_WORKSPACE_CLEANUP__TEMPORARY__PERIOD__MIN+`,"`+180+`","Periodof stopped temporary workspaces cleanup job execution." 
- `+CHE_WORKSPACE_SERVER_PING__SUCCESS__THRESHOLD+`,"`+1+`","Number of sequential successful pings to server after which it is treated as available. Note: the property is common for all servers e.g. workspace agent, terminal, exec etc." 
+ `+CHE_WORKSPACE_CLEANUP__TEMPORARY__INITIAL__DELAY__MIN+`,"`+5+`","Time period to delay the first execution of temporary workspaces cleanup job." 
+ `+CHE_WORKSPACE_CLEANUP__TEMPORARY__PERIOD__MIN+`,"`+180+`","Time period delay between the termination of one execution and the commencement of the next execution of temporary workspaces cleanup job" 
```

Comment:
`Interval between the end of a run and the start of the next run of the temporary workspaces cleanup job`